### DISCUSSION TITLE
I need help linking my CSS file to my html

### DISCUSSION THREAD

**Discussion by trinipose** [community/community#121132] [Programming Help] [answered]
Hi! I've just started using Github and I tried uploading my code and my CSS is not linked to the HTML! (which works when i first typed it out in Visual Studio Code).
I checked and the name matches the one on the html, i don't know what i've done wrong! (image of the two files in my repository)
Thank you in advance !

https://trinipose.github.io/Prueba-1stDelivery/

<img width="914" alt="Screen Shot 2024-04-25 at 14 20 56" src="https://github.com/community/community/assets/163144806/88dbd4ca-c1a8-4c52-9d9b-c633bc9c6685">

**Comment by ZFPuhi** [comment]
After checking I did see some difference in the ```"``` in question, you could try like this as the css file is in the same directory as your index file:

```<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" type="text/css" href="style.css"> 
</head>
<body>

**Reply by trinipose** [reply]
Tried this (along with other suggestions from the forum) but still it's not working unfortunately :( But thank you for your help regardless!

**Comment by arshadakl** [comment]
![githuhu](https://github.com/community/community/assets/66788460/538f3ce3-0e4e-4618-bb93-264b069474e3)
There's an extra apostrophe in the 'type' section.

**Reply by trinipose** [reply]
fixed it! still it's not working unfortunately :( also tried linking with both a different repository with just the CSS and the link from the CSS on this same repository with no success. But thank you for your help regardless!

**Comment by Bl4ck-H4ck3r** [comment]
<link rel="stylesheet" type="text/css" href="style.css">

Add this code in the head section

**Reply by trinipose** [reply]
HI! thank you for your reply! which code?

**Comment by lennis-dev** [comment, marked as answer]
You can link the file using a relative link. Here is the code: 
```html
<link rel="stylesheet" type="text/css" href="./style.css" />
```
This does not link to your `Style.css` repo but to the css file in the `Prueba-1stDelivery` repo, I hope this helps you.

**Reply by trinipose** [reply]
LENNIS THANK YOU SOOOO MUCH!! it's finally working! True life saver, I had to turn in this submission on tuesday! Hope you have an amazing week

**Reply by ebndev** [reply]
Hi @trinipose! Thank you for posting in the Community. Glad to hear @lennis-dev was able to solve your problem.

Please feel encouraged to mark the comment above as the answer 😁

**Reply by trinipose** [reply]
Done! Sorry I didn't do it before, I'm new to how Community works! Thank you!

**Reply by ebndev** [reply]
No worries :) Welcome!

**Comment by kanetjuh** [comment]
To add your own `.css` file you would have to add the following code in the `<head>` tag
```html
<link rel="stylesheet" type="text/css" href="./path/to/your/file.css" />
```

Hopefully this answers your question, and possibly that from others too!


**_In your case_**
You would have to add the following code (Copy-and-paste)
```html
<link rel="stylesheet" type="text/css" href="./style.css" />
```
